[2025-05-06T16:55:36.589+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_airflow.print_hello scheduled__2025-02-07T00:00:00+00:00 [queued]>
[2025-05-06T16:55:36.603+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_airflow.print_hello scheduled__2025-02-07T00:00:00+00:00 [queued]>
[2025-05-06T16:55:36.604+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-06T16:55:36.811+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): print_hello> on 2025-02-07 00:00:00+00:00
[2025-05-06T16:55:36.817+0000] {standard_task_runner.py:60} INFO - Started process 979 to run task
[2025-05-06T16:55:36.821+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'hello_***', 'print_hello', 'scheduled__2025-02-07T00:00:00+00:00', '--job-id', '522', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmp22e4brpp']
[2025-05-06T16:55:36.823+0000] {standard_task_runner.py:88} INFO - Job 522: Subtask print_hello
[2025-05-06T16:55:36.843+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-06T16:55:36.880+0000] {task_command.py:423} INFO - Running <TaskInstance: hello_airflow.print_hello scheduled__2025-02-07T00:00:00+00:00 [running]> on host afa31441e313
[2025-05-06T16:55:36.971+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_***' AIRFLOW_CTX_TASK_ID='print_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-07T00:00:00+00:00'
[2025-05-06T16:55:36.973+0000] {logging_mixin.py:188} INFO - Hello, Airflow! DAG đang chạy thành công.
[2025-05-06T16:55:36.974+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-06T16:55:36.982+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=hello_***, task_id=print_hello, execution_date=20250207T000000, start_date=20250506T165536, end_date=20250506T165536
[2025-05-06T16:55:37.034+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-06T16:55:37.047+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
